tests/test_all.sh

```diff
@@ -21,6 +21,16 @@
 
 # Note you must use the variable "call" in the for the loop
 
+. $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh gitlfs
+. $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh imagemagick
+. $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh wget
+. $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh git
+. $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh python
+. $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh rclone
+. $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh test_data_clavrx
+. $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh test_data_amsr2
+. $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh test_data_abi_day
```

Comment:
Do we still install test_data_abi_day or is this superseded by test_data_noaa_aws?

Reply:
This has always put the abi data in test data noaa aws, that is where the test scripts point,  right?